projects/arc/src/app/main/main.component.ts

```diff
@@ -63,15 +63,6 @@ export class MainComponent
     },
   ];
 
-  navigate(link: string) {
-    this.router.navigate([link]);
-  }
-
-  toggle() {
-    this.sidebarService.toggle(true, 'right');
-    this.toggleFooter = !this.toggleFooter;
-  }
-
   ngOnInit(): void {
```

Comment:
why removed ?

Reply:
bcos i created the standalone component of this, i removed this from here , added this in header comp